src/services/token.ts

```diff
@@ -34,3 +34,19 @@ export async function getToken(
 		console.log('Input data for getting token is incomplete');
 	}
 }
+
+export const fetchPrices = (
```

Comment:
The name is misleading,
```suggestion
export const fetchPrice = (
```